src/services/user.service.js

```diff
@@ -125,6 +126,10 @@ export default class UserService {
     };
 
     if (isCustomer) {
```

Comment:
Chỗ check điều kiện này sao là role Customer nhưng vẫn bắn ra lỗi "You only  have permission to update status!" vậy a. A xem lại điều kiện rồi test xem oke chưa nhé